committee-code-of-conduct/governance/incident-process.md

```diff
@@ -0,0 +1,111 @@
+# Incident reporting and response process
```

Comment:
This can be a follow-up PR, but we will want to add a metadata header for it to be included on the contrib site:
```
---
title: "Code of Conduct Committee Incident Reporting and Response Process"
weight: 550
aliases: [ "/coc-process" ]
description: |
  Overview of the the Code of Conduct Committee's workflow when receiving and responding
  to an incident report. 
---
```